src/main/java/org/cryptomator/hubcli/Backend.java

```diff
@@ -102,7 +102,15 @@ public HttpResponse<String> grantAccess(UUID vaultId, Map<String, String> member
 
 		public HttpResponse<String> getAccessToken(UUID vaultId) throws IOException, InterruptedException, UnexpectedStatusCodeException {
 			var vaultKeyReq = createRequest("vaults/" + vaultId + "/access-token").GET().build();
-			return sendRequest(httpClient, vaultKeyReq, HttpResponse.BodyHandlers.ofString(StandardCharsets.US_ASCII), 200);
+			try {
+				return sendRequest(httpClient, vaultKeyReq, HttpResponse.BodyHandlers.ofString(StandardCharsets.US_ASCII), 200);
+			} catch (UnexpectedStatusCodeException e) {
+				if (e.status == 449) {
+					throw new UnexpectedStatusCodeException(e.status, "Cryptomator Hub CLI is not setup. Execute the setup command first.");
```

Comment:
Add a new constructor with `Throwable cause`

Reply:
Fixed in [c7a5f3d](https://github.com/cryptomator/hub-cli/pull/8/commits/c7a5f3dc56f5c88567d4829bb679336b5306d671)